tekton/publish-run.yaml

```diff
@@ -142,7 +152,7 @@ spec:
         name: tekton-bucket
     params:
     - name: versionTag
-      value: v0.0.0 # REPLACE with the version you want to release, for nightly releases, we might want to use `vYYYYMMDD-commit`
+      value: v0.4.0 # REPLACE with the version you want to release, for nightly releases, we might want to use `vYYYYMMDD-commit`
```

Comment:
Should we keep this one as `v0.0.0` (to not overwrite the v0.4.0 images by mistake later on)